example-mojaloop-backend/values.yaml

```diff
@@ -214,43 +105,42 @@ account-lookup-mysql:
     ##   replicator: /vault/secrets/mysql-replicator
     ##
     customPasswordFiles: {}
-  primary:
```

Comment:
Why was this removed?

Take note that persistence should always be disabled by default, otherwise default installation fail.

Reply:
same again as previous comment .. this is now set to false.